extension/onboarding/onboarding.css

```diff
@@ -212,10 +221,12 @@ p {
   background-image: url("./images/icon-superhero.svg");
 }
 
+#wakeword-try-it p,
```

Comment:
It would be good to change `try-it` to a class instead of an id, so there doesn't have to be two selectors on each of these.